Backend/index.js

```diff
@@ -929,9 +968,9 @@ app.post("/listings/:id/reviews", verifyToken, async (req, res) => {
       data: {
         content,
         rating,
-        reviewerId,
-        sellerId: parseInt(sellerId),
-        listingId: parseInt(listingId),
+        reviewer: { connect: { id: reviewerId } },
+        seller: { connect: { id: parseInt(sellerId) } },
```

Comment:
what does connect do?

Reply:
The connect is used to link new data to existing records in the database,